test/components/generators/chat/test_openai.py

```diff
@@ -617,6 +639,59 @@ def test_live_run(self):
         assert message.meta["finish_reason"] == "stop"
         assert message.meta["usage"]["prompt_tokens"] > 0
 
+    @pytest.mark.skipif(
+        not os.environ.get("OPENAI_API_KEY", None),
+        reason="Export an env var called OPENAI_API_KEY containing the OpenAI API key to run this test.",
+    )
+    @pytest.mark.integration
+    def test_live_run_with_response_format(self):
+        chat_messages = [ChatMessage.from_user("Describe the 20th Nobel Peace Prize.")]
```

Comment:
Let's use a text from which the model can actually extract all the required attributes.